autowiring/AutowiringDebug.h

```diff
@@ -10,62 +10,64 @@ class AutoPacket;
 class CoreContext;
 
 namespace autowiring {
-namespace dbg {
+  namespace dbg {
```

Comment:
I like formatting fixups, but leave this for another PR.  It makes it harder to review important changes.